doc/hotspot-style.md

```diff
@@ -1056,6 +1056,9 @@ and other supported compilers may not have anything similar.
 
 ### Additional Permitted Features
 
+* `alignof`
+([n2341](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2007/n2341.pdf))
```

Comment:
Should say that `alignof` should be used instead of `std::alignment_of`.

Reply:
Got it, swapped to the more descriptive version to mention this. I'll discuss about the reasons for permitting it little later, right now I'm somewhat swamped by build-dev stuff